core/classes/IssueAssignedTimelineEvent.class.php

```diff
@@ -49,15 +49,18 @@ public function __construct( $p_timestamp, $p_user_id, $p_issue_id, $p_handler_i
 	 * @return string
 	 */
 	public function html() {
+		$t_show_summary = config_get( 'timeline_show_issue_summary' );
+		$t_link = string_get_bug_view_link( $this->issue_id, true, false, $t_show_summary );
```

Comment:
These 2 lines of code are repeated in every html() method of all Timeline event classes.

I suggest instead to define a method that returns properly formatted HTML for the bug link in _TimelineEvent_ class, and call it everywhere you reference $t_link.